### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by khanzadyan**
Corrections and extension of the documentation part. There are several missing things to add.

### CODE REVIEW

**Comment by khanzadyan** [comment]
@drobiazko I've separated the Sailor Hooks into a different file and corrected some links to make the README correct. This is ready to be submitted when we have something to release for the sailor.

The Sailor-hooks.md contains all the text which was initially in the README. I thought it is not correct to leave it there in that half-ready state since the README gets displayed on NPM page.

**Comment by drobiazko** [comment]
@khanzadyan you should link the new page from the README. Even though it's not complete it has some interesting info

**Comment by zubairov** [comment]
@khanzadyan I don't like moving it to the different file. What is the problem leaving it in README?

**Comment by khanzadyan** [comment]
@zubairov I moved it to a different file due to its incompleteness. Whenever we make a new Sailor release whatever we have in the README gets pulled into the NPM page. Currently, it looks not so professional https://www.npmjs.com/package/elasticio-sailor-nodejs

**Comment by zubairov** [comment]
No documentation is much worse than some documentation, please put it back

**Comment by khanzadyan** [comment]
@zubairov I've returned the original text about the Sailor hooks back to the README file.

**Comment by khanzadyan** [comment]
@zubairov, @Nazar910 is now taking over writing the explanation about how the hooks are used.

File: README.md [hunk 1]
```diff
@@ -1,83 +1,124 @@
...
 
+
+
 ## Sailor hooks
 
+### Init hook
+
+```javascript
+/**
+* cfg - config object with credentials and other stuff like datasamples
```
Comment (yarikos) [inline README.md]: I would add a notice like:
`This is the same config as the one passed to `processMessage` method of the trigger or action.`

File: README.md [hunk 2]
```diff
@@ -1,83 +1,124 @@
...
- - TBD - Startup logs, where to find it?
- - TBD - Separate them under different tab in UI
- - TBD - Where to see restart errors?
+```javascript
+/**
+* cfg - config object with credentials and other stuff like datasamples
+*/
+exports.startup = function(cfg) {
+    //do stuff
+    const data = messages.newMessageWithBody({
```
Comment (yarikos) [inline README.md]: Pls remove `messages.newMessageWithBody`,
just return an object with data in the promise.

**Pull request merged**